aten/src/ATen/ExpandUtils.h

```diff
@@ -117,7 +117,7 @@ inline std::vector<Tensor> expand_outplace(TensorList to_expand) {
     } else if (to_expand[i].sizes().equals(sizes)) {
       result[i] = to_expand[i];
     } else {
-      result[i] = to_expand[i].expand(sizes);
+      result[i] = to_expand[i].expand(sizes, true);
```

Comment:
I think that the fact that this code needed to be changed is a good sign that this is not a good solution. That's leaking the JIT logic all over the place, and degrading the user-visible API by **a lot**. Can't this have a good default value?

Reply:
It does have a default value. I am not sure what you are asking. These are the library internal calls where the value needs to be non-default.